src/main/groovy/wooga/gradle/release/ReleasePlugin.groovy

```diff
@@ -60,78 +64,129 @@ import wooga.gradle.releaseNotesGenerator.utils.ReleaseBodyStrategy
  * Example:
  * <pre>
  * {@code
- *     plugins {
- *         id "net.wooga.release" version "0.15.1"
- *     }
- * }
+ *     plugins {*         id "net.wooga.release" version "0.15.1"
+ *}*}
  * </pre>
  */
 class ReleasePlugin implements Plugin<Project> {
 
     private static Logger logger = Logging.getLogger(ReleasePlugin)
 
-    static final String ARCHIVES_CONFIGURATION = "archives"
-    static final String UNTIY_PACK_TASK = "unityPack"
+    static final String GROUP = "Wooga"
+    static final String EXTENSION_NAME = "atlasRelease"
+
     static final String CLEAN_META_FILES_TASK = "cleanMetaFiles"
     static final String SETUP_TASK = "setup"
     static final String RC_TASK = "rc"
+    static final String FINAL_TASK = "final"
+    static final String SNAPSHOT_TASK = "snapshot"
     static final String TEST_TASK = "test"
-    static final String GROUP = "Wooga"
-    static final String EXTENSION_NAME = "atlasRelease"
+    static final String RELEASE_NOTES_DESCRIPTION_TASK = "releaseNotes"
```

Comment:
I would rename this to `GITHUB_RELEASE_NOTES_TASK = "generateGithubReleaseNotes"`